react/game/src/app/screens/Profile/index.js

```diff
@@ -1,7 +1,55 @@
-import React from 'react';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { actionCreators as profileActions } from '@redux/profile/actions';
+import profileTypes from '@types/Profile';
 
-function Profile() {
-  return <h1>My Profile</h1>;
+import ProfileLayout from './layout';
+
+class Profile extends Component {
+  state = {
+    editingProfile: false
+  };
+
+  componentDidMount = () => {
```

Comment:
U must use function in comp lifecycle methods
```suggestion
  componentDidMount() {
```

Reply:
sorry, it was a mistake from a snippet